html-css/week1/nadiia/portfolio/index.html

```diff
@@ -0,0 +1,298 @@
+        <li>Adobe Photoshop - helps A LOT with creating animations.</li>
+      </ul>
+      <p class="undertext plus">
+        I know programming languages, such as HTML and CSS.
+      </p>
+      <p class="art_project" id="project">Projects</p>
+      <section id="project" class="project">
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225357803.png"
+            />
+            <p class="iframesundertext">Survey Form</p></a
+          >
+        </div>
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-tribute-page-project/build-a-tribute-page"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225519905.png"
+            />
+            <p class="iframesundertext">Tribute Page</p></a
+          >
+        </div>
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-technical-documentation-page-project/build-a-technical-documentation-page"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225640025.png"
+            />
+            <p class="iframesundertext">Technical Documentation Page</p></a
+          >
+        </div>
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-product-landing-page-project/build-a-product-landing-page"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225851080.png"
+            />
+            <p class="iframesundertext">Product Landing Page</p></a
+          >
+        </div>
+        <div class="art_project_item">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-personal-portfolio-webpage-project/build-a-personal-portfolio-webpage"
+            ><img
+              src="https://theporodyofcats.wordpress.com/wp-content/uploads/2025/01/d0b7d0bed0b1d180d0b0d0b6d0b5d0bdd0bdd18f_2025-01-12_225959169.png"
+            />
+            <p class="iframesundertext">Personal Portfolio Webpage</p></a
+          >
+        </div>
+        <div class="art_project_item">
+          <a href=""
+            ><img src="" />
+            <p class="iframesundertext">
+              This tile is empty... Surely for future projects :3
+            </p></a
+          >
```

Comment:
Avoid using `img` or `a` tags without their primary attributes. The link for example now would reload the page and take the user in the beginning. While the `img` tag would cause an error in some tests as Google Lighthouse.
```suggestion
        <p class="iframesundertext">This tile is empty... Surely for future projects :3</p>
```